modules/openapi-generator/src/main/resources/kotlin-spring/api.mustache

```diff
@@ -57,19 +55,19 @@ import kotlin.collections.Map
 @RequestMapping("\${api.base-path:<%contextPath%>}")
 <%={{ }}=%>
 {{#operations}}
-class {{classname}}Controller({{#serviceInterface}}@Autowired(required = true) val service: {{classname}}Service{{/serviceInterface}}) {
+interface {{classname}} {
```

Comment:
Changing `api.mustache` to an interface class breaks convention we use elsewhere, and will cause anyone with custom templates to break (as the default `interfaceOnly=false` loads a file that didn't previously exist.